docs/manifests/manifest-7329cd9

```diff
@@ -0,0 +1 @@
+openshift-hosted-aro4:7e29cd9/rhcos-44.81.202004250133-0-azure.x86_64
```

Comment:
This is used in our code. We could generate it via `go generate` if we move image vars to `pkg/util/version`